infra/base-images/base-builder/compile

```diff
@@ -264,6 +289,12 @@ else
   fi
 fi
 
+if [[ ! -z "${STORE_CCACHE-}" ]]; then
+  # Cleanup existing cache.
+  rm -rf $WORK/ccache-cache
+  cp -rf /ccache/cache $WORK/ccache-cache
```

Comment:
rather than copy stuff, can we just use `CCACHE_DIR` directly? 

i.e. `https://github.com/google/oss-fuzz/blob/9f56cf477ee8a0a151864c4f8b395344ebef0908/infra/base-images/base-builder/Dockerfile#L202`

Reply:
(This problem also goes away if we just use `docker commit` instead). 

The container persists across steps if we don't use `--rm` i believe.

Reply:
The point is we need to have the ccache cache in a clean version of the project's docker image, prior to running `compile`.

As far as I can tell in order to do this we either need to (1) download the ccache from somewhere or (2) copy it out after a build and then copy it into a new version of the project's image

Reply:
Got it, let's just use `CCACHE_DIR` here. We can set that to `/workspace/ccache` in the cloudbuild steps to pull this out. 

We can also add a simple step in between to clean this up.